spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/SimpleJpaQuery.java

```diff
@@ -45,8 +46,8 @@ final class SimpleJpaQuery extends AbstractStringBasedJpaQuery {
 	 * @param valueExpressionDelegate must not be {@literal null}
 	 */
 	public SimpleJpaQuery(JpaQueryMethod method, EntityManager em, @Nullable String countQueryString,
-			QueryRewriter queryRewriter, ValueExpressionDelegate valueExpressionDelegate) {
-		this(method, em, method.getRequiredAnnotatedQuery(), countQueryString, queryRewriter, valueExpressionDelegate);
+			QueryRewriter queryRewriter, ValueExpressionDelegate valueExpressionDelegate, boolean validation) {
+		this(method, em, method.getRequiredAnnotatedQuery(), countQueryString, queryRewriter, valueExpressionDelegate, validation);
```

Comment:
Probably a `QueryValidator` object would make sense instead of a magic boolean. I can handle that during the merge.